configs/11.0/packages/valgrind/sources

```diff
@@ -20,11 +20,11 @@
 #
 
 ATSRC_PACKAGE_NAME="Valgrind"
-ATSRC_PACKAGE_VER=3.11.0
+ATSRC_PACKAGE_VER=3.13.0
 # Don't update the Valgrind revision without updating the VEX revision too.
 # They have to be in sync.
-ATSRC_PACKAGE_REV=15905
-ATSRC_PACKAGE_VEX_REV=3223
+ATSRC_PACKAGE_REV=16446
+ATSRC_PACKAGE_VEX_REV=3398
```

Comment:
If we keep checking out from trunk, the correct revision from vex is 3385, since it is the last revision that was backported to the branch:

```
svn log --stop-on-copy svn://svn.valgrind.org/vex/branches/VEX_3_13_BRANCH
------------------------------------------------------------------------
r3386 | sewardj | 2017-05-31 11:47:53 -0300 (Wed, 31 May 2017) | 2 lines

Merge, from trunk, r3385 (Handle x86 CET prefixes (32 bit only)).
[...]
```